ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -744,6 +744,27 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure an Administrator Password Is Required to Access System-Wide Preferences (FDA Required)
+  platforms: macOS
+  platform: darwin
+  description: Checks that an Administrator Password Is Required to Access System-Wide Preferences
+  resolution: |
+    Automated method:
+    Ask your system administrator to deploy an MDM profile that disables apple personalized advertising.
```

Comment:
Copy/paste residue `... apple personalized advertising.`

Reply:
Should be fixed